### DISCUSSION TITLE
Precompiled vs compiling myself

### DISCUSSION THREAD

**Discussion by who-am-i-2080** [Jyers/Marlin#1195] [Q&A] [answered]
Hey guys, im new here and i saw that the last update was like 4 days ago (one of the files were changed 4 days ago) but the last version for the precompiled code was released on june 6 (v1.3.4), what should i do? i dont really know how to compile the code to add my autobedLVL (https://www.amazon.com/dp/B08SJ2RYYZ?psc=1&ref=ppx_yo2_dt_b_product_details) and if i should take the precomliped option, which one is the best?

**Comment by Cal-LF** [comment, marked as answer]
There are still some possible issues with the branch merge. I believe it will be sorted out before another version release.
You can see all the past commits made by clicking on the "Jyres" next to the black dot in the light blue highlighted box.
If you pick the latest release, you should be good to go.

**Reply by who-am-i-2080** [reply]
Hey so i used the 10x10 UBL BL touch .bin file, and i have this direct drive part https://www.thingiverse.com/make:849473, what do they mean by setting offsets and how can i do it?

**Reply by mmaravillo** [reply]
This video from CHEP https://www.youtube.com/watch?v=TroPjdkSEOY is one of many that helped me setup bed leveling when I was new to JyersUI.  Set offsets for 1) distance of your probe from the hotend; 2) Z-axis.
You can use gcodes from https://github.com/Jyers/Marlin/discussions/1223 for your slicer.
